index.bs

```diff
@@ -643,12 +656,34 @@ The {{VRDisplay}} associated with this event.
 {{VRDisplayEventReason}} describing why this event has has been fired.
 
 
-## Window Interface extension ## {#interface-window}
+## VREventHandlers Interface ## {#interface-vreventhandlers}
 
 <pre class="idl">
-partial interface Window {
+[NoInterfaceObject]
+interface VREventHandlers {
   attribute EventHandler onvrdisplayconnect;
   attribute EventHandler onvrdisplaydisconnect;
+  attribute EventHandler onvrdisplaynavigate;
+};
+</pre>
+
+User agents implementing this specification MUST provide the following new DOM events. The corresponding events must be of type {{VRDisplayEvent}} and must fire on the {{vr}} object. Registration for and firing of the events must follow the usual behavior of DOM4 Events.
+
+<dfn event for="VREventHandlers" id="vreventhandlers-vrdisplayconnect-event">onvrdisplayconnect</dfn>
+A user agent MAY dispatch this event type to indicate that a {{VRDisplay}} has been connected.
```

Comment:
We need to define more what this means. We've been having issues with the expectations around this event. For instance, a Gear VR is technically always connected, but there is also the USB drop in connection and the mounted indicators. We don't want people to wait to hear a `vrdisplayconnect` event before they `requestPresent` or otherwise try to use VR.

Now that we have Rift, Gear and DayDream, let's define what we think this means for all 3. Rift is kind of easy since maybe it means you've either plugged it in via USB or that you've told the browser you want to be in VR available mode (this is the toggle I've been proposing for the UI so someone could say, please don't enable VR experiences right now I don't want a page to automatically use my Rift).